### DISCUSSION TITLE
Document about experimental features

### DISCUSSION THREAD

**Discussion by yahu1031** [foss42/apidash#8] [General]
I'm trying to make a few changes in the UI and contribute to this project. But suddenly the feature `records` were being used with this code. Kindly please document these kinds of things in the [README.md](https://github.com/foss42/api-dash/blob/main/README.md)/[CONTRIBUTING.md](https://github.com/foss42/api-dash/blob/main/CONTRIBUTING.md). So that things will be easy with development.

**Comment by animator** [comment]
We will document it. 
Thanks for reporting 👍

**Comment by animator** [comment]
[CONTRIBUTING.md](https://github.com/foss42/api-dash/blob/main/CONTRIBUTING.md) has been updated.
